homeassistant/components/wallbox/__init__.py

```diff
@@ -0,0 +1,147 @@
+"""The Wallbox integration."""
+import asyncio
+from datetime import timedelta
+import logging
+
+import requests
+from wallbox import Wallbox
+
+from homeassistant import exceptions
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_PASSWORD, CONF_USERNAME
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.update_coordinator import DataUpdateCoordinator
+
+from .const import CONF_CONNECTIONS, CONF_ROUND, CONF_SENSOR_TYPES, CONF_STATION, DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+PLATFORMS = ["sensor"]
+UPDATE_INTERVAL = 30
+
+
+class WallboxHub:
```

Comment:
This can inherit `DataUpdateCoordinator`. Then we don't need to create new methods that mimic the existing coordinator methods.